microhapdb/tests/test_cli.py

```diff
@@ -109,7 +109,7 @@ def test_main_marker_noargs(capsys):
     microhapdb.cli.main(args)
     out, err = capsys.readouterr()
     outlines = out.strip().split('\n')
-    assert len(outlines) == 1 + 198 + 15 + 40 + 26 + (11 - 1) + 10 + 118 + 90
+    assert len(outlines) == 1 + 198 + 15 + 40 + 26 + (11 - 1) + 10 + 118 + 90 + 25
```

Comment:
There were 25 new haplotypes added, so we add a `+25` here

Reply:
Lol, glad you were able to decipher this.